assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -84,18 +85,25 @@ function MyStories() {
   const [status, setStatus] = useState(STORY_STATUSES[0].value);
   const [typeaheadValue, setTypeaheadValue] = useState('');
   const [viewStyle, setViewStyle] = useState(VIEW_STYLE.GRID);
+  const [currentPageToFetch, setCurrentPageToFetch] = useState(1);
```

Comment:
the `setCurrentPageToFetch` could clamp the value between `0 - totalPages` for safety. we could add this `util` If you wanted. I end up using it a lot:
```
export const clamp = (v: number, range: Range): number => {
  const lowerBound = Math.min(range[0], range[1]);
  const upperBound = Math.max(range[0], range[1]);

  return Math.min(Math.max(lowerBound, v), upperBound);
};
```

Reply:
cool idea, i think this will work nicely with your other suggestions in play. thank you!